src/fides/api/ctl/migrations/versions/54102bba36de_add_attentive_connector.py

```diff
@@ -0,0 +1,74 @@
+"""add_attentive_connector
+
+Revision ID: 54102bba36de
+Revises: 68a518a3c050
+Create Date: 2023-03-09 22:08:17.480643
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "54102bba36de"
+down_revision = "68a518a3c050"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # Add to ConnectionType enum
+    op.execute("alter type connectiontype rename to connectiontype_old")
+    op.execute(
+        "create type connectiontype as enum('postgres', 'mongodb', 'mysql', 'https', "
+        "'snowflake', 'redshift', 'mssql', 'mariadb', 'bigquery', 'saas', 'manual', "
+        "'email', 'manual_webhook', 'timescale', 'fides', 'sovrn', 'attentive')"
```

Comment:
We could also drop "email" from connectiontype here since we're removing it from the enum.  And we can remove "email" connectionconfigs corresponding, but these are unlikely to exist because they've never been exposed in the UI

Reply:
I'll add it just to be safe
```
op.execute("delete from connectionconfig where connectiontype in ('email')")
```